habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -201,12 +217,105 @@ class TeleportActionConfig(ActionConfig):
 class VelocityControlActionConfig(ActionConfig):
     type: str = "VelocityAction"
     # meters/sec
-    lin_vel_range: List[float] = field(default_factory=lambda: [0.0, 0.25])
-    # deg/sec
-    ang_vel_range: List[float] = field(default_factory=lambda: [-10.0, 10.0])
+    lin_vel_range: List[float] = field(default_factory=lambda: [0.0, 0.3])
+    # rad/sec
+    ang_vel_range: List[float] = field(default_factory=lambda: [-0.45, 0.45])
+    # rad/sec
+    ang_vel_range_camera_pitch: List[float] = field(
+        default_factory=lambda: [-0.45, 0.45]
+    )
+    # rad
+    ang_range_camera_pitch: List[float] = field(
+        default_factory=lambda: [-1.57, 0.43]
+    )
+    time_step: float = 0.1  # seconds
+    enable_scale_convert: bool = True
+
+
+@dataclass
+class WaypointControlActionConfig(VelocityControlActionConfig):
+    r"""
+    In Navigation tasks only, this action will utilize a velocity controller to
```

Comment:
That would be great to mention with what types of robot this parameters work and that it enables/suitable for sim2real.